mlrun/db/httpdb.py

```diff
@@ -3888,8 +3888,8 @@ def patch_model_endpoint(
         attributes_keys = list(attributes.keys())
         attributes["name"] = name
         attributes["project"] = project
-        attributes["function-name"] = function_name or None
-        attributes["function-tag"] = function_tag or None
+        attributes["function_name"] = function_name or None
```

Comment:
is this related?

Reply:
No, but it is new bug that were introduced [here](https://github.com/mlrun/mlrun/pull/7668), and also was BP to 1.8.x